signac_dashboard/dashboard.py

```diff
@@ -105,9 +123,32 @@ def __init__(self, config={}, project=None, modules=[]):
                 f"{self.config['CARDS_PER_ROW']}."
             )
 
+        self.config.setdefault("ACCESS_TOKEN", secrets.token_urlsafe())
+
         # Create and configure the Flask application
         self.app = self._create_app(self.config)
 
+        # Initialize the login manager
+        self.login_manager = flask_login.LoginManager()
+        self.login_manager.init_app(self.app)
+
+        @self.login_manager.user_loader
+        def user_loader(identifier):
+            if self.config["ACCESS_TOKEN"] is None:
+                return User("None")
```

Comment:
Must the `id` be a string? If not, I'd prefer `User(None)`.

Reply:
Yes, the user id must be a `str`: https://flask-login.readthedocs.io/en/latest/#your-user-class
> Note that this must be a [str](https://docs.python.org/3/library/stdtypes.html#str) - if the ID is natively an [int](https://docs.python.org/3/library/functions.html#int) or some other type, you will need to convert it to [str](https://docs.python.org/3/library/stdtypes.html#str).

Reply:
Thanks! 👍 Sorry, I hadn’t looked at the flask-login docs at all.